src/libraries/System.Text.Json/tests/Common/ReferenceHandlerTests/ReferenceHandlerTests.IgnoreCycles.cs

```diff
@@ -236,6 +236,11 @@ public async Task IgnoreCycles_OnExtensionData()
         [Fact]
         public async Task IgnoreCycles_DoesNotSupportPreserveSemantics()
         {
+            if (StreamingSerializer is null)
```

Comment:
Is there a condition on the `Fact` we could do instead? That would describe the platforms/configuration this test isn't supported in.

Reply:
I'm not sure, it's an abstract test suite and `StreamingSerializer` is overridden by derived classes. I don't think it could be done with the existing `ConditionalFact`/`ConditionalTheory` infrastructure without substantial refactoring of the test suites.